streams-runtimes/streams-runtime-local/src/main/java/org/apache/streams/local/tasks/StreamsPersistWriterTask.java

```diff
@@ -146,7 +146,7 @@ public void run() {
             try {
                 Thread.sleep(5000);
             } catch (InterruptedException e) {
-                LOGGER.error("Sleep interrupted: {}", e);
+                //This is the expected behavior when this thread is killed
```

Comment:
I wrestled with that when I stood up this PR, since this feels like a bit of a do-nothing fix. My understanding was that when this was written, the sleep was necessary in order to give all of the writer components time to shut themselves down. Otherwise, we would get ourselves into a situation where race conditions are possible.

This is obviously not ideal design, but at the same time, I'm not sure we're looking to drastically refactor the local runtime since it is working. What are your thoughts?